bag_verifier/src/main/resources/application.conf

```diff
@@ -4,5 +4,5 @@ aws.outgoing.sns.topic.arn=${?outgoing_topic_arn}
 aws.metrics.namespace=${?metrics_namespace}
 bag-verifier.primary-storage-bucket=${?primary_storage_bucket_name}
 operation.name=${?operation_name}
-bag-verifier.mode.is_replica=${?is_replica}
+bag-verifier.mode=${?bag_verifier_mode}
```

Comment:
Do you need the azure endpoint?

Reply:
yes. the PR is not ready yet. I have to fix a couple of things